client/src/pages/reports/Form.js

```diff
@@ -133,16 +145,26 @@ export default class ReportForm extends ValidatableFormWrapper {
 		</HelpBlock>
 
 		const {ValidatableForm, RequiredField} = this
+		const submitText = currentUser.hasActivePosition() ? 'Preview and submit' : 'Save draft'
+		const alertStyle = {top:132, marginBottom: '1rem', textAlign: 'center'}
 
-		const submitText = currentUser.hasAssignedPosition() ? 'Preview and submit' : 'Save draft'
-
+		const supportEmail = dict.lookup('SUPPORT_EMAIL_ADDR')
+		const supportEmailMessage = supportEmail ? `(${supportEmail})` : ''
+		const warningMessageNoPosition = `You do not have an active position and cannot submit a report, please contact the support team ${supportEmailMessage} and request them to set your position status to active`
```

Comment:
I would change this to `You do not have an active position and cannot submit a report, please contact your organization's super users and request them to set your position status to active. If you are unsure, you can also contact the support team ${supportEmailMessage}`
@dspCarlson can you take a look?

Reply:
My suggestion: You cannot submit a report.  Your assigned advisor position has an inactive status.  Please contact your organization's super users {some magic to identify the superusers} and request that they set your position status to active.